Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -1354,28 +1380,6 @@ private extension ChatMessageListVC {
         listView.contentOffset.y = newOffset
     }
 
-    // If we are inserting messages at the bottom, update the previous cell
-    // to hide the timestamp of the previous message if needed.
-    func reloadPreviousMessageWhenInsertingNewMessage() {
```

Comment:
nit: We could have remained with the functions but changed them to return the indexPaths instead of performing the reload. For example, `indexPathForPreviousMessageWhenInsertingNewMessage() -> IndexPath?`

But not a big deal

Reply:
I'll consider it